src/features/chart/components/Chart.tsx

```diff
@@ -53,17 +54,22 @@ const formatDate = (isoDate, ChartRange) => {
 export const Chart = () => {
   const dispatch = useDispatch();
   const { selectedChartData, selectedChartRange } = useSelector(
-    state => state.chartData
+    (state: any) => state.chartData
   );
-  const { selectedStockTicker } = useSelector(state => state.stockTickerData);
-  const updateChartRange = stock => dispatch(updateChartAction(stock));
-  const onClickHandler = event => {
-    updateChartRange(event.target.value);
+  const { selectedStockTicker } = useSelector(
+    (state: any) => state.stockTickerData
+  );
+  const updateChartRange = (stock: any) =>
```

Comment:
And here, with `stock` - No `any` typing please, where you can get away with it.